### PR TITLE
added html, css and agent stub for view counter

### PR DESCRIPTION
**Pull request by charliew-rsi**
This branch contains styling for a view counter and a stub in the agent for the click tracking that will be implemented on the new back-end

### CODE REVIEW

File: src/components/ArticlePreview.js [hunk 1]
```diff
@@ -33,6 +38,11 @@ const ArticlePreview = props => {
     }
   };
 
+  const onHandleLinkClickTrack = event => {
```
Comment (wesdollar) [inline src/components/ArticlePreview.js]: This isn't doing anything other than persisting an event, which we do nothing with from what I can tell. If code is dead, remove it.

File: src/components/ArticlePreview.js [hunk 2]
```diff
@@ -52,11 +62,11 @@ const ArticlePreview = props => {
           </button>
         </div>
       </div>
-
-      <Link to={`/article/${article.slug}`} className="preview-link">
+      <span className="viewsCounter">This article has {views} views</span>
+      <Link to={`/article/${article.slug}`} onClick={onHandleLinkClickTrack} className="preview-link">
```
Comment (wesdollar) [inline src/components/ArticlePreview.js]: Remove onClick if onHandleLickClickTrack has no functionality.

File: src/constants/actionTypes.js [hunk 3]
```diff
@@ -33,4 +33,5 @@ export const UPDATE_FIELD_EDITOR = 'UPDATE_FIELD_EDITOR';
 export const FOLLOW_USER = 'FOLLOW_USER';
 export const UNFOLLOW_USER = 'UNFOLLOW_USER';
 export const PROFILE_FAVORITES_PAGE_UNLOADED = 'PROFILE_FAVORITES_PAGE_UNLOADED';
-export const PROFILE_FAVORITES_PAGE_LOADED = 'PROFILE_FAVORITES_PAGE_LOADED';
\ No newline at end of file
+export const PROFILE_FAVORITES_PAGE_LOADED = 'PROFILE_FAVORITES_PAGE_LOADED';
+export const ARTICLE_ADDVIEW = 'ARTICLE_ADDVIEW';
```
Comment (wesdollar) [inline src/constants/actionTypes.js]: Make sure you're ending including a blank line at the end of all your files. In Javascript, there are specific reasons for doing this, which I encounrage you to google. Make sure eslint and prettier are enforcing these standards.

File: src/components/ArticlePreview.js (hunk 1 above)
Comment (charliew-rsi) [reply src/components/ArticlePreview.js]: Removing. Will add on another branch after finishing middleware and back-end services for handling the click tracking.

File: src/components/ArticlePreview.js (hunk 2 above)
Comment (charliew-rsi) [reply src/components/ArticlePreview.js]: Removing. Will add on another branch after finishing middleware and back-end services for handling the click tracking.

File: src/constants/actionTypes.js (hunk 3 above)
Comment (charliew-rsi) [reply src/constants/actionTypes.js]: Added blank line at end of file.

**Pull request merged**